### ISSUE TITLE
Rule disable mean no auto-renew of certificate

### ISSUE THREAD

**Issue by mcnahum** [NginxProxyManager/nginx-proxy-manager#412]
if you have a proxy host entry who is disable, the auto-renew of the certificate is not working ...

**Comment by jc21** [comment]
Yes this is a known problem and can cause some warning messages but is not a show stopper. You can manually renew the certificate after you re-enable the host that's using it.

**Comment by mcnahum** [comment]
it will be good to have it in the log ... is it something who can be discussed with the NGinX team? I guess you have more access to them than us...
